src/map/constants.inc.h

```diff
@@ -0,0 +1,970 @@
+	script->constdb_comment("Status Icons");
```

Comment:
this is being used to support constants names in db/clans.conf.
Should I remove it? So I need to change the function in clan.c and the db/clans.conf to receive Icon ID instead of icon name?